searx/webapp.py

```diff
@@ -41,10 +41,12 @@
     logger.critical("cannot import dependency: pygments")
     from sys import exit
     exit(1)
+
 try:
-    from cgi import escape
-except:
     from html import escape
+except ImportError:
+    from cgi import escape
```

Comment:
Its not about dropping py2 .. its about implement python 3 correctly (without exceptions) .. why should we prefer a implementation that is focused on old obsolete py2 and raise exceptions on new py3 instances?

Reply:
Does it solve an issue with the Google engines ?

There is the same pattern for https://github.com/asciimoo/searx/blob/master/searx/utils.py : Python 2 as default and then switch to Python 3 in case of exception.

There is already https://github.com/asciimoo/searx/issues/1471 to drop Python 2 support.

Reply:
Thanks for the info .. I won't revert b44d0d90 & 433c639d / we really have other problems.